engine/src/main/java/com/datatorrent/stram/StreamingAppMasterService.java

```diff
@@ -138,6 +139,7 @@
    * This should be replaced when a constant is defined there
    */
   private static final String SSL_SERVER_KEYSTORE_LOCATION = "ssl.server.keystore.location";
+  private static int NODE_MANAGER_KILL_CONTAINER_TIMEOUT;
```

Comment:
How about a more appropriate name as this timeout is not a Node Manager setting but rather a fallback setting. You could just call it REMOVE_CONTAINER_TIMEOUT (after the RM call removeContainerRequest)